app/templates/views/guidance/features/roadmap.html

```diff
@@ -58,6 +58,10 @@ <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
 
   <ul class="govuk-list govuk-list--bullet">
 
+    <li>Let services attach pages to letter templates</li>
+    <li>Give organisations permission to approve new services</li>
+    <li>Users can consent to take part in research</li>
+    <li>Introduce subheadings for emails</li>
```

Comment:
```suggestion
    <li>Introduce subheadings for emails</li>
```

Link to https://www.notifications.service.gov.uk/using-notify/formatting

or https://www.notifications.service.gov.uk/using-notify/formatting#headings-and-subheadings if we think it’s ok to deep link to an anchor